buildkite/bazelci.py

```diff
@@ -1819,9 +1884,26 @@ def main(argv=None):
             )
         elif args.subparsers_name == "runner":
             configs = fetch_configs(args.http_config, args.file_config)
+            tasks = configs.get("tasks", {})
```

Comment:
Here's the place where we could nicely do:

```
tasks = configs.get("tasks", configs.get("platforms", {}))
```

Reply:
no longer needed since I changed the config method